browser/brave_content_browser_client.cc

```diff
@@ -115,6 +117,40 @@ bool HandleURLRewrite(GURL* url,
   return false;
 }
 
+bool HandlePDFjsURLRewrite(GURL* url,
+                           content::BrowserContext* browser_context) {
+  if (url->SchemeIs(extensions::kExtensionScheme) &&
+      url->DomainIs(pdfjs_extension_id) && url->has_query()) {
+    return true;
+  }
+
+  return false;
+}
+
+bool HandlePDFjsURLReverseRewrite(GURL* url,
+                                  content::BrowserContext* browser_context) {
+  // Remove the PDF.js prefix from display URLs unless that's all there is.
+  if (url->SchemeIs(extensions::kExtensionScheme) &&
+      url->DomainIs(pdfjs_extension_id) && url->has_query()) {
```

Comment:
nit: I would move url->has_query() down to the next line to make it easier to read it as 3 `&&` conditions